module/VuFind/src/VuFind/Db/Entity/ExchangeArrayTrait.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Entity model for user table
```

Comment:
Maybe this would be more accurate:

```suggestion
 * Trait providing a basic implementation of ExchangeArrayInterface.
```